src/registries/menus/menu_items_actions.ts

```diff
@@ -623,7 +619,6 @@ export const CREATE_CHART = (env: SpreadsheetChildEnv) => {
       dataSets,
       labelRange,
       type: "bar",
-      background: BACKGROUND_CHART_COLOR,
```

Comment:
You should do the same in `chart_factory.ts`, the other place we create charts

Reply:
To help you, you can look for all `BACKGROUND_CHART_COLOR` in the code to see where it's used.

Reply:
On xlsx export, helper method applies default background on chart if nothing is specified. Should we keep this behavior or alter it as well? rest all suggested changes are done

Reply:
For xlsx you can keep as it

Reply:
> rest all suggested changes are done

I do not see changes in `chart_factory.ts`

Reply:
Please correct me if I am wrong, I have checked all the files with `BACKGROUND_CHART_COLOR`, I believe the changes you are referring too are in `chart_factory` master branch and not in 16.0

Reply:
Yes you are right. We should not forget to update it during the forward-port :)